libs/commons/Cache_disk.mli

```diff
@@ -35,6 +35,13 @@ val cache :
   'input ->
   'value
 
+(* Need unit to avoid value restriction in impl. would be nice to not need somehow? *)
```

Comment:
if we were using eio instead of lwt we would not need all those _lwt functions. We would have just one and it would just work.

Reply:
soon!